clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -10266,35 +10288,13 @@ Value *CodeGenFunction::EmitAArch64SVEBuiltinExpr(unsigned BuiltinID,
 
 Value *CodeGenFunction::EmitAArch64SMEBuiltinExpr(unsigned BuiltinID,
                                                   const CallExpr *E) {
-  // Find out if any arguments are required to be integer constant expressions.
```

Comment:
The changes to EmitAArch64SMEBuiltinExpr seem like a (significant) non-functional change. Can you separate it out as such so that it can be committed separately?